signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/view/SignInWithAppleButton.kt

```diff
@@ -63,51 +99,66 @@ class SignInWithAppleButton @JvmOverloads constructor(
         }
     }
 
-    override fun onDetachedFromWindow() {
-        super.onDetachedFromWindow()
-        service = null
-        dialog = null
+    fun configure(service: SignInWithAppleService, client: SignInWithAppleClient) {
+        this.service = service
+        this.client = client
+
+        configureFragmentIfCreated()
+    }
+
+    private fun configureFragmentIfCreated() {
+        val service = service
+        val client = client
+
+        if (service == null || client == null) {
+            return
+        }
+
+        val fragmentIfCreated = fragmentTag?.let {
+            client.getFragmentManagerForSignInWithApple().findFragmentByTag(fragmentTag)
+                    as? SignInWebViewDialogFragment
+        }
+
+        fragmentIfCreated?.configure(service, client)
```

Comment:
This method can be called both from the setter of `fragmentTag` and from `configure`, because it is a function of the three things that change in those two places, and they happen in one order or the other depending on whether there is a rotation involved.